modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/Registry.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model.dag
+
+import org.tessellation.ext.kryo.KryoRegistrationId
+import org.tessellation.schema.transaction.RewardTransaction
+
+import eu.timepit.refined.numeric.Interval
+
+object Registry {
+
+  import eu.timepit.refined.auto._
+
+  type RunnerKryoRegistrationIdRange = Interval.Closed[300, 399]
```

Comment:
interval overlaps with existing range

Reply:
Please elucidate this comment for me.

Reply:
Kryo can be used with or without pre-registration. We require it for security. You register a given class under numeric identifier which is used then for deserialization - hence, identifiers must be unique. You cannot serialize two different classes with the same identifier as you will not be able to deserialize it.
It's simple to keep everything clean when you have one module and one registration map, but in the project we have many modules and all of the registrations are combined into single one and single kryo pool.
That being said, we have to somehow manage these identifiers not to duplicate anything. 
See https://github.com/Constellation-Labs/tessellation/blob/develop/doc/adr/0009-kryo-class-registration-policy.md for the kryo class registration policy we use currently. Interval 300-399 is reserved exclusively for `shared` module, so you cannot reuse the same identifiers here, as they will be duplicated which will cause (de)serialization issues.

Nevertheless, for now you don't need kryo for the rosetta as there is nothing that currently uses it. You need kryo for two cases:
- when you need a hash of an object,
- when you need a binary format of an object.

I believe that rosetta classes will neither be required in binary format, nor hashed, so you don't need to register classes in kryo at all. You can (and for sure you will need) kryo pool, to get the hash of a transaction or a block, but these classes are already registered in their respective modules.

Reply:
This file is not currently needed and may not be needed anymore. It has been removed.